subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -673,7 +673,7 @@ void *radio_pkt_decrypt_get(void)
 /* Dedicated Rx PDU Buffer for Control PDU independent of node_rx with BIS Data
  * PDU buffer
  */
-static uint8_t pkt_big_ctrl[sizeof(struct pdu_big_ctrl)];
+static uint8_t pkt_big_ctrl[sizeof(struct pdu_bis)];
```

Comment:
it would be good to explain why we use the size of a pdu_bis structure, and not the pdu_big_ctrl structure

Reply:
The buffer use is now clarified in the comment. I hope you like it :)